src/Compilers/CSharp/Portable/Binder/Binder_Operators.cs

```diff
@@ -1634,6 +1648,57 @@ static bool isNuint(TypeSymbol type)
             }
         }
 
+        private UnaryOperatorAnalysisResult? UnaryOperatorExtensionOverloadResolution(
+            UnaryOperatorKind kind,
+            BoundExpression operand,
+            CSharpSyntaxNode node,
+            BindingDiagnosticBag diagnostics,
+            out LookupResultKind resultKind,
+            out ImmutableArray<MethodSymbol> originalUserDefinedOperators)
+        {
+            resultKind = LookupResultKind.Empty;
+            originalUserDefinedOperators = [];
+
+            if (operand.IsLiteralDefault() || // Reported not being able to target-type `default` elsewhere, so we can doing more work
```

Comment:
nit: Do we need this line? (seems the second condition subsumes that)

Reply:
`ReportUnaryOperatorError` checks for this, and I do not think we we want to do any work in this case as well.